### PR TITLE
Fix broken/redirected links

### PR DESCRIPTION
**Pull request by Josh-Cena**
- Google redirected privacy-sandbox URLs again
- Fix some anchor links
- Fix a few broken internal/external links

### CODE REVIEW

File: files/en-us/web/api/reporting_api/index.md [hunk 1]
```diff
@@ -96,9 +96,9 @@ These HTTP response headers define the endpoints where reports are sent.
 
 - {{HTTPHeader("Reporting-Endpoints")}}
   - : Sets the name and URL of reporting endpoints.
-    These can be used in the `report-to` directive, which may be used with a number of HTTP headers including {{httpheader("Content-Security-Header")}}.
+    These can be used in the `report-to` directive, which may be used with a number of HTTP headers including {{httpheader("Content-Security-Policy")}}.
```
Comment (hamishwillee) [inline files/en-us/web/api/reporting_api/index.md:99]: Hmmm. I hope I didn't inject this. I fixed a few of these last week, but it is possible I either missed or added these.

**Review by hamishwillee — APPROVED** [review]
Looks good! Thanks very much.

File: files/en-us/web/api/reporting_api/index.md (hunk 1 above)
Comment (Josh-Cena) [reply files/en-us/web/api/reporting_api/index.md:99]: Haha. I think you wrote it 😄 I was meaning to tag you asking whether it's intentional.

Comment (hamishwillee) [reply files/en-us/web/api/reporting_api/index.md:99]: Clearly I'm trying to hide that fact :-0. I fixed one case in https://github.com/mdn/content/pull/35602 which has not merged yet. Thanks for fixing these.

**Pull request merged**